usage-examples/go/atlas-sdk-go/internal/metrics/disk.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/atlas-sdk/v20250219001/admin"
+)
+
+// FetchDiskMetrics returns measurements for a specified disk partition
+func FetchDiskMetrics(
+	ctx context.Context,
+	sdk admin.MonitoringAndLogsApi,
+	p *admin.GetDiskMeasurementsApiParams,
+) (*admin.ApiMeasurementsGeneralViewAtlas, error) {
+
```

Comment:
Nit: we're inconsistent about leading newlines after func declarations. This file and `process.go` in this dir both contain leading newlines after the func declarations; the other files do not have leading newlines after the func declaration. We should probably remove these in the files in this dir.

Reply:
blegh. that was to help me scan and compare across files, and I incorrectly assumed running IDE `reformat code` would reset it.